src/main/java/org/hyperskill/hstest/checker/CheckLibraryVersion.java

```diff
@@ -0,0 +1,131 @@
+package org.hyperskill.hstest.checker;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.*;
+import java.lang.reflect.Type;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.time.LocalDate;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+* Checks if the current version of the library is the latest one on GitHub releases page of the library.
+* If not, throws an exception.
+ */
+public class CheckLibraryVersion {
+
+    private String VERSION_FILE = "src/main/java/org/hyperskill/hstest/resources/version.txt";
+
+    private String currentVersion;
+    private String latestVersion;
+    private boolean isLatestVersion = true;
+
+    public CheckLibraryVersion() {
+    }
+
+    /**
+     * Checks if the current version of the library is the latest one on GitHub releases page of the library.
+     * If not, throws an exception.
+     */
+    public void checkVersion() {
+        LocalDate lastChecked = null;
+        File lastCheckedFile = new File("LastChecked.txt");
+        if (lastCheckedFile.exists()) {
+            try {
+                FileReader fileReader = new FileReader(lastCheckedFile);
+                BufferedReader reader = new BufferedReader(fileReader);
+                lastChecked = LocalDate.parse(reader.readLine());
+                reader.close();
+            } catch (IOException e) {
+                throw new RuntimeException(e);
```

Comment:
Why this method contains a lot of RuntimeException's? That's not a good way of handling checked exceptions making them unchecked exceptions. At the very least they should be UnexpectedError's.